tools/ds-identify

```diff
@@ -960,15 +960,27 @@ dscheck_LXD() {
     if is_socket_file /dev/lxd/sock; then
         return ${DS_FOUND}
     fi
-    # On LXD KVM instances, /dev/lxd/sock is not yet setup by
-    # lxd-agent-loader's systemd lxd-agent.service.
-    # Rely on DMI product information that is present on all LXD images.
-    # Note "qemu" is returned on kvm instances launched from a host kernel
-    # kernels >=5.10, due to `hv_passthrough` option.
-    # systemd v. 251 should properly return "kvm" in this scenario
-    # https://github.com/systemd/systemd/issues/22709
-    if [ "${DI_VIRT}" = "kvm" -o "${DI_VIRT}" = "qemu" ]; then
```

Comment:
Why drop the virt check?

Reply:
Ahh now I see your point. I thought you were implying the entire section I had removed. You are right it should be gated by a conditional asserting we are kvm or qemu.